.changeset/brave-timers-walk.md

```diff
@@ -0,0 +1,5 @@
+---
+'@conform-to/valibot': patch
+---
+
+fix: support nested fields with only checkboxes
```

Comment:
Nitpick: I think this is more like a feature / minor. But up to you :)

Reply:
To be honest, I was worried about what to do. 😅
I just wanted to make a patch release based on past [pull requests](https://github.com/edmundhung/conform/pull/733).